polaris-react/src/components/IndexTable/IndexTable.scss

```diff
@@ -282,6 +324,17 @@ $loading-panel-height: 53px;
     var(--p-duration-100);
 }
 
+.TableHeadingSortIcon-heading-align-end {
+  transition: width var(--p-duration-200) var(--p-ease-in) var(--p-duration-100);
+  overflow: hidden;
+  width: 0;
```

Comment:
Same as above can we animate more performant values then `width`?

Reply:
I was able to achieve the same overall animated effect without the icon manipulation.